sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_generated/_patch.py

```diff
@@ -6,9 +6,119 @@
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import List
+from typing import List, Optional, Any, Union
+from azure.core import PipelineClient
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+from azure.core.pipeline import policies
 
-__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+from azure.appconfiguration._generated import models as _models
+from azure.appconfiguration._generated._azure_app_configuration import (
+    AzureAppConfiguration as AzureAppConfigurationGenerated,
+)
+from azure.appconfiguration._generated._configuration import (
+    AzureAppConfigurationConfiguration as AzureAppConfigurationConfigurationGenerated,
+    VERSION,
+)
+from azure.appconfiguration._generated._serialization import Deserializer, Serializer
+
+
+class AzureAppConfiguration(AzureAppConfigurationGenerated):
+    """AzureAppConfiguration.
+
+    :param credential: Credential needed for the client to connect to Azure. Required.
```

Comment:
Another option here is playing with the `__doc__` of the parent, instead of copying everything over. This way if the generated doc ever changes, you don't need to update your patch doc as well

Reply:
Is it possible to update part and use the rest from parent doc? I need to update the credential type.